dependencies.gradle

```diff
@@ -47,7 +47,7 @@ ext {
     navigationVersionName = "1.0.0-alpha3"
     testingVersionName = "1.0.0-alpha4"
     snackbarVersionName = "1.0.0-alpha2"
-    navigationRailVersionName = '1.0.0-beta1'
+    navigationRailVersionName = '1.0.0-alpha2'
```

Comment:
We can use alpha01, this will be the first time we release this library.

Reply:
Fixed